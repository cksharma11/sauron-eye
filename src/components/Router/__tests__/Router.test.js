import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import App from "../Router";

configure({ adapter: new Adapter() });

describe("App", () => {
  it("should render properly", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
