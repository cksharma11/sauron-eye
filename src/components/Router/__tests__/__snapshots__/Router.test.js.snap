// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render properly 1`] = `
"<HashRouter history={{...}}>
  <Route exact={true} path=\\"/\\" component={[Function: bound Dashboard]} />
  <Route path=\\"/history\\" component={[Function: bound History]} />
</HashRouter>"
`;
